### PR TITLE
Add fix to threshold validation and change error message

### PR DESCRIPTION
**Pull request by Gubbsy**
Add fix to high spend model to account for spends equal to the threshold value. Amend the error message to reflect this.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -969,6 +969,7 @@ en-GB:
             amount:
               not_a_number: "Enter amount of spend"
               greater_than: "Enter a spend greater than £500"
+              greater_than_or_equal_to: "Enter a spend greater than or equal to the spend threshold"
             vat_amount:
```
Comment (ptrelease) [inline config/locales/en.yml:973]: Lets discuss if we can use the "Enter a spend greater than £500" content.  And replace £500 with a dynamic string.

Comment (Gubbsy) [reply config/locales/en.yml:972]: Discussed, and confirmed with Abbie. This content is fine.

**Review by ptrelease — APPROVED** [review]

**Comment by ptrelease** [comment]
Waiting on Product decision on whether to allow greater than or equal to.

**Comment by ptrelease** [comment]
Product Owner has signed off on proposed fix.

**Pull request merged**